src/client/components/ResultMessageModalWindow/ResultMessageModalWindow.styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.ResultMessageModalWindow-component {
+  margin-top: 20%;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  border: 1px solid black;
```

Comment:
You actually do not need these css from line 9 to 13. 
display:Flex => It is not needed because your component doesn't need any another container along it side by side. It is only used for that, right.
Flex-direction => Not needed
align-content: center=> You are using it like two times for your content one in this class and second in the container-div, so doesn't makes any sense.
border: 1px solid black => There is no border in the reference design.
I am so sorry but these changes are important and because of this extra css you have a scroll in your storybook which is not a good sign for responsiveness.

Reply:
its repaired ...